### PR TITLE
update eslint depdendencies to latest

### PR DESCRIPTION
**Pull request by Paulooze**
Updated eslint in every package - had to make some breaking changes in `typescript-eslint` config: 
  - https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/ban-ts-comment.md
  - https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md

### CODE REVIEW

File: eslint-config-innovatrics-typescript-base/index.js [hunk 1]
```diff
@@ -102,6 +106,17 @@ module.exports = {
     // might be situations where we must use an interface,
     // but those cases seem to be rare (and we will just disable
     // the rule for that line of code)
-    '@typescript-eslint/consistent-type-definitions': ['error', 'type'],
-  }
+    "@typescript-eslint/consistent-type-definitions": ["error", "type"],
+    "@typescript-eslint/explicit-module-boundary-types": "off",
```
Comment (dhlavaty) [inline eslint-config-innovatrics-typescript-base/index.js]: pls explain in comment

File: eslint-config-innovatrics-typescript-base/index.js [hunk 2]
```diff
@@ -102,6 +106,17 @@ module.exports = {
...
+    "@typescript-eslint/naming-convention": [
```
Comment (dhlavaty) [inline eslint-config-innovatrics-typescript-base/index.js:111]: probably should be ABOVE "the line"

File: eslint-config-innovatrics-typescript-base/index.js [hunk 3]
```diff
@@ -102,6 +106,17 @@ module.exports = {
...
+      "error",
+      {
+        selector: "default",
+        format: ["camelCase", "UPPER_CASE", "PascalCase"],
+        leadingUnderscore: "allow",
+      },
+      { selector: "typeLike", format: ["PascalCase"] },
+    ],
+    "@typescript-eslint/ban-ts-comment": 2,
```
Comment (dhlavaty) [inline eslint-config-innovatrics-typescript-base/index.js:122]: pls explain in comment

File: eslint-config-innovatrics-typescript/index.js [hunk 4]
```diff
@@ -122,6 +127,17 @@ module.exports = {
     // might be situations where we must use an interface,
     // but those cases seem to be rare (and we will just disable
     // the rule for that line of code)
-    '@typescript-eslint/consistent-type-definitions': ['error', 'type'],
-  }
+    "@typescript-eslint/consistent-type-definitions": ["error", "type"],
+    "@typescript-eslint/explicit-module-boundary-types": "off",
```
Comment (dhlavaty) [inline eslint-config-innovatrics-typescript/index.js]: pls explain in comment

File: eslint-config-innovatrics-typescript/index.js [hunk 5]
```diff
@@ -122,6 +127,17 @@ module.exports = {
...
+    "@typescript-eslint/naming-convention": [
```
Comment (dhlavaty) [inline eslint-config-innovatrics-typescript/index.js:111]: probably should be above "the line"

File: eslint-config-innovatrics-typescript/index.js [hunk 6]
```diff
@@ -122,6 +127,17 @@ module.exports = {
...
+      "error",
+      {
+        selector: "default",
+        format: ["camelCase", "UPPER_CASE", "PascalCase"],
+        leadingUnderscore: "allow",
+      },
+      { selector: "typeLike", format: ["PascalCase"] },
+    ],
+    "@typescript-eslint/ban-ts-comment": 2,
```
Comment (dhlavaty) [inline eslint-config-innovatrics-typescript/index.js:122]: pls explain in comment

**Review by dhlavaty — CHANGES_REQUESTED** [review]

File: eslint-config-innovatrics-typescript-base/index.js [hunk 7]
```diff
@@ -32,68 +32,69 @@ module.exports = {
...
-        consistent: true
-      }
+        consistent: true,
+      },
     ],
 
     // Dusan hates this one
-    'no-lonely-if': 'off',
+    "no-lonely-if": "off",
+    "no-else-return": "off",
```
Comment (gabor) [inline eslint-config-innovatrics-typescript-base/index.js:56]: please add "// Gabor does not like this one" before the no-else-return :-]

**Review by gabor — CHANGES_REQUESTED** [review]

**Review by gabor — APPROVED** [review]

File: eslint-config-innovatrics/index.js [hunk 8]
```diff
@@ -90,29 +93,31 @@ module.exports = {
...
+      },
+    ],
 
     // dusan hates this
-    'react/destructuring-assignment': 'off',
+    "react/destructuring-assignment": "off",
 
     // We will not write propTypes anymore, as they will be discontinued at all by Facebook
     "react/prop-types": "off",
     "react/no-unused-prop-types": "off",
```
Comment (dhlavaty) [inline eslint-config-innovatrics/index.js:121]: explain in comment pls

File: eslint-config-innovatrics/index.js [hunk 9]
```diff
@@ -90,29 +93,31 @@ module.exports = {
...
-    'react/require-default-props': "off",
+    "react/require-default-props": "off",
```
Comment (dhlavaty) [inline eslint-config-innovatrics/index.js:124]: explain in comment pls

**Review by dhlavaty — CHANGES_REQUESTED** [review]

**Review by dhlavaty — APPROVED** [review]

**Pull request merged**